addons/payment_buckaroo/const.py

```diff
@@ -10,3 +10,17 @@
     'refused': (690,),
     'error': (490, 491, 492,),
 }
+
+# The currencies supported by Buckaroo, in ISO 4217 format.
+# See https://support.buckaroo.eu/frequently-asked-questions
+# Last seen online: 7 November 2022.
+SUPPORTED_CURRENCIES = [
+    'EUR',
+    'GBP',
+    'PLN',
+    'DKK',
+    'NOK',
+    'SEK',
+    'CHF',
+    'USD',
```

Comment:
Since we are changing those files, let's use recent providers as an example and put all the currencies on the same line (until the 100 char limit is reached ofc)